### ISSUE TITLE
Note empty values and x,y df

### ISSUE THREAD

**Issue by kelshmo** [Sage-Bionetworks/dccvalidator-app#43]
These are suggestions more than issues: 

- Note empty columns with a warning in place of a success message. For example, Successes: "All biospecimen columns present" might also be associated with Warning: "Missing organ, tissue,BrodmannArea values". 

- Be descriptive with x and y noting "missing from biospecimen file" and "missing from manifest"
<img width="479" alt="Screen Shot 2019-09-03 at 3 34 58 PM" src="https://user-images.githubusercontent.com/40647130/64213343-803ad680-ce61-11e9-9b34-6400de2021aa.png">

thanks for this tool!

**Comment by karawoo** [comment]
Good ideas! In the future would you please split multiple bugs/feature requests into separate issues? It helps with tracking and prioritizing work, and makes it easier to automatically close issues with a single PR that addresses them. Thanks!

**Comment by kelshmo** [comment]
yes, will do.

**Comment by karawoo** [comment]
Point 2 is addressed in dccvalidator [here](https://github.com/Sage-Bionetworks/dccvalidator/commit/4f0cefeaa3da9571a088b4c30d01e4ab33e78d25) (not yet reflected on the live version of the app though)

**Comment by karawoo** [comment]
Question @kelshmo -- do you think there should be warnings if _any_ values are missing in the column? Or only if the column is completely empty? The former seems potentially useful but I expect it would result in many spurious warnings.

**Comment by kelshmo** [comment]
Since we are validating against required variables, I actually cannot think of a use case where there *should* be partially missing values. However, this logic does break down if a user is uploading different data types within a single manifest. 

This could be an additional use case for a warning and not an error - "variableX is missing 23 observations". Adding an additional warning does not necessarily make validation stricter, rather encourages greater compliance to a schema.

**Comment by karawoo** [comment]
Yes, in AMP-AD we often have cases where multiple data types are uploaded with a single manifest.  It's not just data either -- if they are uploading the data + metadata files with the manifest, the metadata files won't have the same annotations as the data.

I was thinking we'd also perform this check for the metadata files that are uploaded, and some columns in those templates are not applicable in all cases.

**Comment by kelshmo** [comment]
Sure - in that case, to not overwhelm the user, we should not provide warnings for partially missing values.

**Comment by kelshmo** [comment]
Alternatively, it is extra information that may be helpful in some cases. There are pros and cons :)

**Comment by karawoo** [comment]
Maybe let's try it with just completely empty columns to start, and see what the user feedback is?

**Comment by kelshmo** [comment]
sounds great!
